### PR TITLE
Add way to set a min strength metter

### PR DESCRIPTION
**Pull request by GeoloeG-IsT**

### CODE REVIEW

File: gold-password-input-strength-meter.html [hunk 1]
```diff
@@ -138,7 +138,7 @@
             type: Boolean
           },
           /**
-           * Contains the current score value of the input.
+           * Contains the current label score value of the input.
            */
           _strengthMeterScore: {
             type: String,
```
Comment (GeoloeG-IsT) [inline gold-password-input-strength-meter.html]: Let's rename this to `_strengthMeterScoreLabel` then.

File: gold-password-input-strength-meter.html [hunk 2]
```diff
@@ -153,7 +153,15 @@
               warning: 'None',
               suggestions: 'None'};
             }
-          }
+          },
+          /**
+           * Contains the current score value of the input.
+           */
+          _score: Number,
```
Comment (GeoloeG-IsT) [inline gold-password-input-strength-meter.html]: and this one as `_strengthMeterScore`

File: gold-password-input-strength-meter.html [hunk 3]
```diff
@@ -153,7 +153,15 @@
...
+          /**
+          * Min strength metter (used on validate).
+          */
+          minStrengthMetter: String,
```
Comment (GeoloeG-IsT) [inline gold-password-input-strength-meter.html]: A typo all over your changes `Metter` -> `Meter` and therefore, for the sake of consistency, let's call it `minStrengthMeterScoreLabel`

**Comment by GeoloeG-IsT** [comment]
@RoXuS , merci / thanks for you implementation, let's work together on the associated PR if you don't mind!

**Comment by RoXuS** [comment]
@MeTaNoV, I don't mind, I have made the changes. Let my know if you want any others changes.

**Comment by GeoloeG-IsT** [comment]
@RoXuS , merci Julien pour les MaJ!

**Pull request merged**